test/unit/test_il_vm.c

```diff
@@ -205,9 +205,9 @@ static bool test_rzil_vm_op_let() {
 	mu_assert_notnull(vars, "vars vector");
 	mu_assert_eq(rz_pvector_len(vars), 0, "cleanup");
 	rz_pvector_free(vars);
-
+	
```

Comment:
remove this tab.

Reply:
I ran clang format but got  this errors(almost on all the files):  Error reading My_Rizin_Dev/rizin/.clang-format: Invalid argument

My_Rizin_Dev/rizin/.clang-format:27:1: error: unknown key 'RequiresClausePosition'

RequiresClausePosition: SingleLine

Reply:
be sure to have the right version.

Reply:
What version of clang/clang-format is the right version?

Reply:
16. https://github.com/rizinorg/rizin/blob/dev/DEVELOPERS.md#c